simulation-scripts/perturb.sh

```diff
@@ -134,7 +134,18 @@ run_scenario() {
 
   copy_challenge_and_tasks "${SCENARIO_DIR}"
 
+  cleanup
+}
+
+cleanup() {
+  shopt -u nullglob
+  if ls "${TMP_DIR}"/perturb-script-file-* > /dev/null 2>&1; then
```

Comment:
Could we just do a test of the shell expansion being non-empty and avoid the ugly redirections and the extra child process (ls)